pages/_app.tsx

```diff
@@ -1,9 +1,60 @@
 import 'normalize.css/normalize.css';
 import '../global.scss';
 import '../icomoon-v1.0/style.css';
-
+import { DefaultSeo } from 'next-seo';
 import type { AppProps } from 'next/app';
+import Head from 'next/head';
+
+const meta = {
+  title: 'Polski Front-End',
+  description: `Serwis Polski Front-End powstał w celu zebrania w jednym miejscu jak największej liczby stron, serwisów oraz blogów na temat szeroko rozumianego front-end developmentu. Co ważne, wszystkie zgromadzone tutaj serwisy tworzone są w języku polskim!`,
+};
 
 export default function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <DefaultSeo
+        title={meta.title}
+        description={meta.description}
+        openGraph={{
+          type: 'website',
+          title: meta.title,
+          locale: 'pl_PL',
+          url: `https://${process.env.NEXT_PUBLIC_URL!}`,
+          description: meta.description,
+          images: [
+            {
```

Comment:
Ten obrazek średnio działa z Facebookiem. Trzeba zrobić inny.

Reply:
https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fpolskifrontend-7m75f85k3.vercel.app%2F

Reply:
Z tego co widzę, to w oryginale jest specjalny obrazek pod og image. Przerzucę go